Resources.md

```diff
@@ -0,0 +1,118 @@
+BLS signatures are more space-efficient but more computation-intensive and security assumption is looser.
+If you want to go deeper a [implementation of BLS signature algorithms](https://medium.com/cryptoadvance/bls-signatures-better-than-schnorr-5a7fe30ea716) have been made the Chia team.
+
+2-party aggregated pubkeys are also workable on [ECDSA](https://eprint.iacr.org/2017/552.pdf) but far harder to implement.
+Further discussion on the LN ml about [2p ECDSA](https://lists.linuxfoundation.org/pipermail/lightning-dev/2018-April/001221.html)
+
+## Implementations
+
+Schnorr is [under implemenation](https://github.com/bitcoin-core/secp256k1/pull/558) in bitcoin-core ECC library, libsecp256k1.
+You can find a [bip-schnorr rust implementation](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2018-November/016506.html) of some applications
+like aggregated sigs, accountable sigs and threshold sigs.
+
+## Further improvement
+
+If Schnorr is deployed, we may build on top further applications like [cross-input signature aggregation](https://bitcointalk.org/index.php?topic=1377298.0), [non-interactive signature aggregation per-block](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2017-May/014272.html), [generalised taproot](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2018-July/016249.html) (more efficient taproot in non-cooperative case).
+
+## Lightning
+
+Lightning will reap a huge bonus of seeing Schnorr adoption on the base layer. To go through a simple [introduction](https://jonasnick.github.io/blog/2018/09/04/schnorr-and-taproot-in-lightning/).
+
+More in-depth:
+* how to eliminate payment hash correleation ? Switch to [payment-points/scalar locks](https://lists.launchpad.net/mimblewimble/msg00086.html) and [theoritical discussion](https://eprint.iacr.org/2018/472.pdf)
+* how to enable [Atomic Multi-Payment path will keeping the Proof-of-Payment](https://lists.linuxfoundation.org/pipermail/lightning-dev/2018-March/001100.html) ?
+* what interactions between [LN-invoices and scriptless scripts](https://lists.linuxfoundation.org/pipermail/lightning-dev/2018-November/001489.html) ?
+* how to enable [stuckless payments](https://suredbits.com/payment-points-part-2-stuckless-payments/) without resorting to script-update on the whole network or use sha256 midstate ?
+* what's about more expressive e2e contracts like [escrow](https://lists.linuxfoundation.org/pipermail/lightning-dev/2019-June/002028.html) or even [combination of payment-points type of contract](https://lists.linuxfoundation.org/pipermail/lightning-dev/2019-October/002213.html)
+
+A quick skim on how of lightings tx [could look like](https://lists.linuxfoundation.org/pipermail/lightning-dev/2019-May/001996.html) and even further ideas to redesign ligthning txns in a [post-schnorr world](https://lists.linuxfoundation.org/pipermail/lightning-dev/2018-February/001031.html)
```

Comment:
s/of lightnings tx/about how Lightning Network transactions/ and s/ligthning/Lightning (or) LN/